### DISCUSSION TITLE
Assets not found when running / copying examples

### DISCUSSION THREAD

**Discussion by ambi88dex** [bevyengine/bevy#11302] [Q&A] [answered]
As mentioned in the documentation below:
https://github.com/bevyengine/bevy/blob/main/examples/shader/shader_material.rs#L29

I'm trying to run this example.
The problem is I'm unable to source the image for this!
Can anyone share where find the source for this image?

**Comment by alice-i-cecile** [comment, marked as answer]
You need to copy the correct files from the assets directory into your project. In this case, https://github.com/bevyengine/bevy/blob/v0.12.1/assets/branding/icon.png
